src/components/percentWheel/PercentWheel.module.scss

```
@@ -0,0 +1,13 @@
+.PercentWheel {
+  position: relative;
+
+  >.value {
+    transform: translate(-50%, -100%);
+    top: 50%;
+    left: 50%;
+    position: absolute;
```

When resizing the view horizontally smaller, the words become bigger than the chart.
If you end up using my grid suggestion above(below?) then I found this to work
```suggestion
    position: absolute;
    @media screen and (min-width: breakpoint(medium)) and (max-width: breakpoint(large)) {
      font-size: rem(10);
    }
```